src/ast.js

```diff
@@ -247,8 +247,21 @@ var operatorMap = {};
     operatorMap[TOK.T_PERCENT] = Sk.astnodes.Mod;
 }());
 
+function py3_operators(py3) {
```

Comment:
I don't think we should be modifying this operatorMap. We could just check in `getOperator` if it's available when we get the operator right?

Reply:
Well, you need to indicate somewhere that @ is or is not an arithmetic operator.  Currently, ```operatorMap``` is where that is done.  Yes, we could modify ```getOperator``` instead, but then it would need to have a notion of whether the operator found in ```operatorMap``` is valid or not.  Requiring another structure to indicate that somehow.  In other words, I believe this is exactly the structure that tells you whether or not the operator is "available", though I would have used the word "valid".

By doing it this way, you update the operator map *once* in ```Sk.configure``` based on whether you are using Python2 or Python3, then ```getOperator``` works correctly.  If we move a check into ```getOperator``` it would have to run that check every time an operator is parsed.

If you have a better way, can you be more concrete about what you mean?

Reply:
FWIW I'm in favour of @rixner's approach here, although I'd prefer a more descriptive name like `setupPy3Operators` to make it more clear that it's a one-time init function.

In fact, my only comment is on introducing the `matmul` function in `type.js` -- I imagine we'll pick up more python-3-specific dunder-methods as we go, so we should go ahead and create a `setupPython3DunderMethods` (pick a name as you prefer).

Otherwise, this change looks awesome!